src/app/static/src/app/store/modelCalibrate/mutations.ts

```diff
@@ -100,8 +98,8 @@ export const mutations: MutationTree<ModelCalibrateState> = {
     },
 
     [ModelCalibrateMutation.SetComparisonPlotData](state: ModelCalibrateState, action: PayloadWithType<any>) {
```

Comment:
The type of the payload can be more specific now that you've regenerated the types to include the comparison plot data type.